client/my-sites/plans-features-main/index.tsx

```diff
@@ -150,6 +150,8 @@ export interface PlansFeaturesMainProps {
 	hideUnavailableFeatures?: boolean; // used to hide features that are not available, instead of strike-through as explained in #76206
 	showLegacyStorageFeature?: boolean;
 	isSpotlightOnCurrentPlan?: boolean;
+	// Can pass array of features that will be bolded and pushed to top of feature list
+	highlightedFeatures: string[];
```

Comment:
This property seems redundant now. We can call the hook directly from `use-plan-features-for-grid-plans.ts` (might need to pass `isInSignup` there instead).

Reply:
Good idea. Simpler. This has been done.